### PR TITLE
[onert] LinearExecutor Notifies Execution{Op,Subgraph}EndEvent

### PR DESCRIPTION
**Pull request by glistening**
Let LinearExecutor notify events.

ONE-DCO-1.0-Signed-off-by: Sanggyu Lee <sg5.lee@samsung.com>

Related: https://github.com/Samsung/ONE/issues/10604
3rd commit from Inner Repo's Draft PR
It requires #10607.

### CODE REVIEW

**Comment by glistening** [comment]
Another build error on android. I am using ndk r21, but CI uses r20 which is more strict. I will set up `r20/ndk` and check it.

File: runtime/onert/core/src/exec/LinearExecutor.cc [hunk 1]
```diff
@@ -49,9 +49,26 @@ void LinearExecutor::executeImpl()
...
+      auto op_idx = code.op_ind;
+      auto &backend_ctx = _backend_contexts.at(backend);
+      const auto &tensor_reg = backend_ctx->tensor_registry;
+      const auto &op = _graph.operations().at(op_idx);
+      const auto &outputs = op.getOutputs();
+      if (outputs.size() != 1)
+        throw std::runtime_error("Only 1 output operator is supported for recording minmax.");
+      auto out_tensor = tensor_reg->getITensor(outputs.at(0));
+      auto opcode = code.op->opcode();
+      notifyExecuteOpEnd(subg_idx, op_idx, opcode, out_tensor);
```
Comment (glistening) [inline runtime/onert/core/src/exec/LinearExecutor.cc:63]: NOTE: `notifyExecuteOpEnd` signature should be changed for multiple outputs. https://github.com/Samsung/ONE/pull/10607#issuecomment-1499893984

**Comment by glistening** [comment]
I've found Android NDK [`r20b`](https://github.com/android/ndk/wiki/Unsupported-Downloads#r20b), not exactly `r20`.

**Comment by glistening** [comment]
Just for recording

> $ ndk/20.0.5594570/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android29-clang++ --version
Android (5220042 based on r346389c) clang version 8.0.7 (https://android.googlesource.com/toolchain/clang b55f2d4ebfd35bf643d27dbca1bb228957008617) (https://android.googlesource.com/toolchain/llvm 3c393fe7a7e13b0fba4ac75a01aa683d7a5b11cd) (based on LLVM 8.0.7svn)
Target: aarch64-unknown-linux-android29
Thread model: posix

r20 uses clang 8.0.7

> $ /home/gyu/bin/ndk-r21/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android29-clang++ --version
Android (5900059 based on r365631c) clang version 9.0.8 (https://android.googlesource.com/toolchain/llvm-project 207d7abc1a2abf3ef8d4301736d6a7ebc224a290) (based on LLVM 9.0.8svn)
Target: aarch64-unknown-linux-android29
Thread model: posix

while r21 uses clang 9.0.8.

**Comment by glistening** [comment]
I've fixed android build and make it supports multiple outputs signature.
But `tracing_ctx` is beyond this PR. Let's handle in #10617.

**Comment by hseok-oh** [comment]
@glistening Could you check my internal draft and give your opinion?

**Comment by hseok-oh** [comment]
@nnfw-bot test onert-x64-release

**Comment by glistening** [comment]
@hseok-oh I will close this PR and make new PRs which uses existing execution event system.

**Pull request closed without merging**